Resources/Audio/Effects/attributions.yml

```diff
@@ -236,3 +236,13 @@
   copyright: 'created by waveplaySFX on Freesound'
   license: "CC0-1.0"
   source: https://freesound.org/people/waveplaySFX/sounds/553744/
+
+- files: [gravity_jump.ogg]
```

Comment:
I would rather not include this additional sound file if it is not even used. It only increases the game's download size. If it is needed in the future it can be added then.